tests/php/AnnotationSessionTest.php

```diff
@@ -114,16 +114,16 @@ public function testGetVolumeFileAnnotationsHideOwnImage()
             'hide_other_users_annotations' => false,
         ]);
 
-        $annotations = $session->getVolumeFileAnnotations($image, $ownUser);
+        $yieldAnnotations = $session->getVolumeFileAnnotations($image, $ownUser);
```

Comment:
I tested it but still get an error, because the arrays have not the same ordering.

Reply:
The ordering is important for the `contains()` method. Since the generator is used, the arrays have a different ordering than before which leads to failing tests. So using `iterator_to_array()` does not solve the problem. That's why I changed the assertions.

Reply:
It's not the ordering of the array keys that's important but the fact that the model objects are not converted to arrays any more. Before, the models were converted to arrays with this:
```php
$annotations = collect($annotations->toArray());
```
Now we need this:
```php
$annotations = collect(collect($annotations())->toArray());
```
This works without further changes to the assertions.

Reply:
Btw: `collect()` also seems to expand the generator, so `iterator_to_array()` is not needed after all.